src/net.cpp

```diff
@@ -3814,14 +3814,10 @@ bool CConnman::DisconnectNode(NodeId id)
     return false;
 }
 
-void CConnman::RelayTransaction(const CTransaction& tx)
+void CConnman::RelayTransaction(const CTransaction& tx, const bool is_dstx)
```

Comment:
I think instead adding here a flag `is_dstx` better to make 2 interface methods: RelayTransaction / RelayCJTransaction to avoid confusion such as using CJManager in EHF Manager (which know nothing about CJ)

Reply:
I think `RelayTransaction` is fine as it is, `is_dstx` is not optional so there isn't scope for a function accidentally flagging a transaction as not CoinJoin.

But I do agree that since the EHF Manager should have no knowledge of CoinJoin and as it is crafting the transaction itself, can simply set `is_dstx` to `false`.